io/include/traccc/io/csv/make_measurement_edm.hpp

```diff
@@ -0,0 +1,28 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "traccc/definitions/primitives.hpp"
+#include "traccc/edm/measurement.hpp"
+
+// Local include(s).
+#include "traccc/io/csv/measurement.hpp"
+
+#pragma once
+
+namespace traccc::io::csv {
+
+/// Make measurement EDM from csv measurement
+///
+/// @param[in] csv_meas input csv measurement
+/// @param[in] acts_to_detray_id Map for acts-to-detray geometry ID converision
+///
+traccc::measurement make_measurement_edm(
```

Comment:
Does this need to be a public header? If this function is only used as part of this library, let's move this header under `src/`. Then I can be a lot more forgiving about using an obscure `std::map` argument in its interface. 🤔